src/data-api.js

```diff
@@ -108,9 +108,6 @@ function dataAPI(ClientRDSDataAPI, Client, dialect) {
       if (obj.method === 'insert') {
         if (dialect === 'mysql') {
           obj.response = [obj.response.insertId];
-          // The data-api client is returning single fields in an object
```

Comment:
This seems to be a different behaviour compared to the local postgres. Maybe something changed when updating the dependencies?